server_cmds_test.go

```diff
@@ -75,15 +75,19 @@ func TestCmdsServer(t *testing.T) {
 			return
 		}
 		expected := &Server{
-			Status:                                 "template",
+			Status:                                 "online",
 			MaxClients:                             32,
 			Name:                                   "Test Server",
+			Uptime:                                 6802178,
```

Comment:
This is a time in seconds not nanoseconds

Reply:
See the explanation above.